Cargo.toml

```diff
@@ -107,6 +107,7 @@ frame-executive = { git = "https://github.com/paritytech/polkadot-sdk.git", tag
 frame-system-benchmarking = { git = "https://github.com/paritytech/polkadot-sdk.git", tag = "polkadot-v1.9.0", default-features = false }
 frame-system-rpc-runtime-api = { git = "https://github.com/paritytech/polkadot-sdk.git", tag = "polkadot-v1.9.0", default-features = false }
 frame-try-runtime = { git = "https://github.com/paritytech/polkadot-sdk.git", tag = "polkadot-v1.9.0", default-features = false }
+frame-metadata = { git = "https://github.com/paritytech/frame-metadata.git", default-features = false }
```

Comment:
*(question from author)*

Should I keep this in case we want to give another shot at proper client-side errors?